### PR TITLE
fix(barchartcard): add classname to top container, allow empty series prop

### LINKED ISSUE #1684: [BarChartCard] classname prop is not attached to the top container
### What package is this for?

- [x] React
- [ ] Angular

### To Reproduce

1. Go to BarChartCard story
2. Inspect container element
3. See that className prop is not attached to the top container

### Expected behavior

Should be attached to the top container

### LINKED ISSUE #1685: [BarChartCard] if series is empty, it fails to render the empty state
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

If series is empty in the `isEditable` state, the card fails to render. This use case was brought up in the new DashboardEditor. When a user first adds a new card, there is no series defined, so there is no sample dataset to be generated from.

### To Reproduce

1. Go to BarChartCard
2. Change `series` to empty array
3. Change `isEditable` to true

### Expected behavior

Should show empty card state

### PR DESCRIPTION
**Pull request by sstone2423**
Closes #1685 
Closes #1684 

**Summary**

- `className` prop is now added to the top container in BarChartCard
- If `series` is an empty array, show the empty card state
- If `values` is empty, do not generate colors, as theres nothing to add colors to

**Change List (commits, features, bugs, etc)**

- Updated tests
- Updated stories
- Added `values: null` to defaultProps to be more explicit

**Acceptance Test (how to verify the PR)**

- Add a custom className and see that its attached to the top BarChartCard container
- Set `series: []` and `isEditable: true` and see that the empty state renders

### CODE REVIEW

File: src/components/BarChartCard/BarChartCard.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import SimpleBarChart from '@carbon/charts-react/bar-chart-simple';
 import StackedBarChart from '@carbon/charts-react/bar-chart-stacked';
 import GroupedBarChart from '@carbon/charts-react/bar-chart-grouped';
-import classnames from 'classnames';
+import classNames from 'classnames';
```
Comment (JoelArmendariz) [inline src/components/BarChartCard/BarChartCard.jsx:5]: Is there a reason to change this?

**Review by JoelArmendariz — APPROVED** [review]
LGTM

File: src/components/BarChartCard/BarChartCard.jsx (hunk 1 above)
Comment (sstone2423) [reply src/components/BarChartCard/BarChartCard.jsx]: Nope, this was a mistake. Fixed it

**Review by scottdickerson — APPROVED** [review]
LGTM

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**